applications/nrf_desktop/src/hw_interface/Kconfig.motion

```diff
@@ -51,34 +51,42 @@ config DESKTOP_MOTION_SENSOR_TYPE
 	default "pmw3360" if DESKTOP_MOTION_SENSOR_PMW3360_ENABLE
 	default "paw3212" if DESKTOP_MOTION_SENSOR_PAW3212_ENABLE
 
-config DESKTOP_MOTION_UP_KEY_ID
+config DESKTOP_MOTION_BUTTONS_UP_KEY_ID
```

Comment:
We could go through procedure for deprecated Kconfig options, but it would introduce more work and it could be slightly more complicated for maintenance. I wonder if it's actually worth it, because the build error (together with proper release note) would quite clearly indicate what's wrong and how to fix it.

Reply:
Let's make an exception here as you suggested. The simulated motion module should not be used in the production system as it is used with our DKs to demonstrate some of the functionalities of the nRF Desktop project.

Reply:
Also, changing this for existing configurations should be straightforward and build error is easy to spot.